src/storage-client/src/types/errors.rs

```diff
@@ -411,6 +411,15 @@ pub enum DataflowError {
 
 impl Error for DataflowError {}
 
+mod columnation {
+    use crate::types::errors::DataflowError;
+    use timely::container::columnation::{CloneRegion, Columnation};
+
+    impl Columnation for DataflowError {
+        type InnerRegion = CloneRegion<DataflowError>;
```

Comment:
`DataflowError` does contain pointers. Why is it okay to use a `CloneRegion` for it?

Reply:
I'll add a justification, but the TL;DR is that it's correct, it's complicated to implement columnation (recursive type, wide enums), and we accept the performance penalty, which should be on par to what we currently have with storing the errors in vectors. For reference, here is an incomplete attempt to implement `Columnation` for `DataflowError`: https://gist.github.com/antiguru/2608b8bb40f1a948aab0761a4ac9d0e5